app.rb

```diff
@@ -0,0 +1,101 @@
+require './library'
+
+class App
+  attr_reader :library
+
+  def initialize
+    @library = Library.new
+  end
+
+  def list_books
+    books = library.list_books
+    puts 'List of books:'
+    if books.empty?
+      puts 'No books found.'
+    else
+      books.each { |book| puts "#{book.title} by #{book.author}" }
+    end
+  end
+
+  def list_people
+    people = library.list_people
+    puts 'List of people:'
+    if people.empty?
+      puts 'No people found.'
+    else
+      people.each { |person| puts "Name: #{person.name}, Age: #{person.age}, Id: #{person.id} " }
+    end
+  end
+
```

Comment:
- [ ] Great job so far 👏🏻 But, your school library console app must behaves exactly  in the same was as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided for you.
- Check the screen short bellow 👇🏻 


![image](https://github.com/prius2055/oop-school-library/assets/79058364/1549e92b-7b3e-4382-894d-8399005e5107)

- 1 Make sure to respect the first option (1) list_books. You need to specify the Title of the book and the Author of the book. Eg
```ruby
Tile: "...", 
Author: "...."
```
- 1 Make sure to respect the option (1) list_people. You need to specify the  if is the Student or Teacher.
```ruby
[Student] Name: Rose, ...
[Teacher] Name: ....

Reply:
This was an oversight. It has been addressed.Thank you